feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -85,6 +113,10 @@ The below yaml defines the OC paths intended to be covered by this test.  OC pat
 paths:
   ## Config Paths ##
 
+  /network-instances/network-instance/protocols/protocol/bgp/global/config/as:
+  /network-instances/network-instance/protocols/protocol/isis/global/config/instance-id:
```

Comment:
Although these paths are used in the DUT configuration, please remove them from the list here because they are not the focus of this test (afts).

Reply:
done